tools/gen_usb_descriptor.py

```diff
@@ -32,8 +32,26 @@
                     help='devices to include in descriptor (AUDIO includes MIDI support)')
 parser.add_argument('--hid_devices', type=lambda l: tuple(l.split(',')), default=DEFAULT_HID_DEVICES,
                     help='HID devices to include in HID report descriptor')
-parser.add_argument('--msc_num_endpoint_pairs', type=int, default=1,
-                    help='Use 1 or 2 endpoint pairs for MSC (1 bidirectional, or 1 input + 1 output (required by SAMD21))')
+parser.add_argument('--renumber_endpoints', type=int, default=1,
+                    help='use relative(1) or absolute(0) endpoint number')
```

Comment:
Not too important, but a nice style for boolean `argparse` arguments is given here: https://stackoverflow.com/a/15008806/142996

Reply:
Done